src/plugin/scope.ml

```diff
@@ -342,10 +359,10 @@ let get_current_scope t =
   let { module_name; ocaml_name = _; _ } = StringMap.find t.proto_path t.type_db in
   (String.lowercase_ascii module_name) ^ t.proto_path
 
-let get_current_proto_path { proto_path; _ } =
+let get_rpc_path { proto_path; _ } =
```

Comment:
I named it as a protobuf package path, but the modified doesn't return the rpc path.
https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#appendix-a---grpc-for-protobuf

Reply:
Indeed, and Thanks for the link (Ive actually been searching for this). I will rename the endpoint back